cloud-aws-cloudformation/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/connector/resource/AwsUpdateService.java

```diff
@@ -121,4 +136,43 @@ private CloudResource getCloudFormationStack(List<CloudResource> resources) {
         return resources.stream().filter(resource -> ResourceType.CLOUDFORMATION_STACK == resource.getType()).findFirst()
                 .orElseThrow(() -> new NotFoundException("CloudFormation stack is not found, the resource might have been deleted."));
     }
+
+    public void modifyVolumes(AuthenticatedContext authenticatedContext, List<String> volumeIds, String diskType, int size) throws Exception {
+        AmazonEc2Client amazonEC2Client = getEc2Client(authenticatedContext);
+        for (String volume: volumeIds) {
+            LOGGER.debug("Modifying volume {} to disk type {} and size {}", volume, diskType, size);
+            try {
+                ModifyVolumeRequest modifyVolumeRequest = ModifyVolumeRequest.builder().volumeId(volume).volumeType(diskType).size(size).build();
+                amazonEC2Client.modifyVolume(modifyVolumeRequest);
+            } catch (software.amazon.awssdk.services.ec2.model.Ec2Exception ex) {
```

Comment:
please import the class instead of defining it here with package

Reply:
done